pkg/build/types/image_configuration.go

```diff
@@ -101,25 +103,28 @@ func (ic *ImageConfiguration) parse(ctx context.Context, configData []byte) erro
 	return nil
 }
 
-func (ic *ImageConfiguration) maybeLoadRemote(ctx context.Context, imageConfigPath string) error {
+func (ic *ImageConfiguration) maybeLoadRemote(ctx context.Context, imageConfigPath string, configHasher hash.Hash) error {
 	data, err := fetch.Fetch(imageConfigPath)
 	if err != nil {
 		return fmt.Errorf("unable to fetch remote include from git: %w", err)
 	}
 
-	return ic.parse(ctx, data)
+	return ic.parse(ctx, data, configHasher)
 }
 
-// Loads an image configuration given a configuration file path.
-func (ic *ImageConfiguration) Load(ctx context.Context, imageConfigPath string) error {
+// Load - loads an image configuration given a configuration file path.
+// Returns checksum (sha256-) of the loaded config or error.
+func (ic *ImageConfiguration) Load(ctx context.Context, imageConfigPath string, configHasher hash.Hash) error {
```

Comment:
What should callers pass here if they're not interested in locked/hashed configs?

Reply:
Updated the comment about this method.